frontend/src/lib/RunUtils.ts

```diff
@@ -66,6 +60,18 @@ function getPipelineName(run?: ApiRun | ApiJob): string | null {
   return (run && run.pipeline_spec && run.pipeline_spec.pipeline_name) || null;
 }
 
+function getPipelineVersionId(run?: ApiRun | ApiJob): string | null {
+  return run &&
+    run.resource_references &&
+    run.resource_references.some(
+      ref => ref.key && ref.key.type && ref.key.type === ApiResourceType.PIPELINEVERSION,
+    )
+    ? run.resource_references.find(
+        ref => ref.key && ref.key.type && ref.key.type === ApiResourceType.PIPELINEVERSION,
```

Comment:
nit: move this to a util function

Reply:
Did you mean moving this to another util func to be called by this util func?